src/software/git/github/CI.md

```diff
@@ -0,0 +1,79 @@
+# 持续集成介绍
+
+## 概念
+
+持续集成( Continuous Integration )指的是，频繁地(一天多次)将代码集成到主干，每次集成都通过自动化的构建(包括编译，发布，自动化测试)来验证，从而尽早地发现集成错误。
+
+## 目的
+
+让产品可以快速迭代，同时还能保持高质量。
+就是说每完成一个完整的部分，就向下个环节交付，发现问题可以马上调整。使得问题不会放大到其他部分和后面的环节。

```

Comment:
This will render as 

`让产品可以快速迭代，同时还能保持高质量。 就是说每完成一个完整的部分，就向下个环节交付，发现问题可以马上调整。使得问题不会放大到其他部分和后面的环节。` 

in markdown

If you want paragraphs, sperate them with a emply line as I said in docs. Otherwise DO NOT start a new line.